thefuck/rules/cat_dir.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+
+def match(command):
+    return (
+        command.script.startswith('cat') and
+        re.match(r'cat: .+: Is a directory', command.output)
+    )
+
+
+def get_new_command(command):
+    return re.sub(r'^cat', 'ls', command.script)
```

Comment:
This can be easily replaced with [`str.replace`](https://docs.python.org/3/library/stdtypes.html?highlight=replace#str.replace). Just make sure that only the first occurrence of `cat` is replaced.

Reply:
I wouldn't have considered the case of more than one `cat`. I added a test for this as well.